js/tiles.js

```diff
@@ -60,7 +60,7 @@ ImageCollection.prototype.showIn = function(container) {
 ImageCollection.prototype.playerEnters = function(player) {
   this.container.classList.add("current");
 };
-ImageCollection.prototype.playerLeaves = function(player) {
+ImageCollection.prototype.playerLeaves = function(player) {top (copy)
```

Comment:
When I want to play your labyrinth, I get an error: Reference Error copy is not defined.

Could you check that the labyrinth is still playable?